app/components/navigation-bar.js

```diff
@@ -0,0 +1,5 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  tagName: '',
```

Comment:
i made this a tagless component - it simplifies the dom

Reply:
We discussed this a while back and if I remember correctly we decided to not go for `tagName: ''` for the "Classic" implementation. There is already an Octane implementation that makes use of this new way of developing components and it would be very useful for us to show some of the differences between the two 👍 

Can you change this to be: 

```javascript
tagName: 'nav',
classNames: ['navbar', 'navbar-light']
```

and then remove the `<nav>` wrapper from the template that would be great 🎉